src/plugins/theme/fontSize.js

```diff
@@ -1,39 +1,16 @@
 /**
  * Font Size Defaults.
+ * For larger font sizes, use our custom modular typography classes (.su-type-0 to .su-type-6)
  */
 module.exports = function () {
   return {
-    xs: ['0.75rem', { lineHeight: '1rem' }],
-    sm: ['0.875rem', { lineHeight: '1.25rem' }],
-    base: ['1rem', { lineHeight: '1.5rem' }],
-    lg: ['1.125rem', { lineHeight: '1.75rem' }],
-    xl: ['1.25rem', { lineHeight: '1.75rem' }],
-    '2xl': ['1.5rem', { lineHeight: '2rem' }],
-    '3xl': ['1.875rem', { lineHeight: '2.25rem' }],
-    '4xl': ['2.25rem', { lineHeight: '2.5rem' }],
-    '5xl': ['3rem', { lineHeight: '1' }],
-    '6xl': ['3.75rem', { lineHeight: '1' }],
-    '7xl': ['4.5rem', { lineHeight: '1' }],
-    '8xl': ['6rem', { lineHeight: '1' }],
-    '9xl': ['8rem', { lineHeight: '1' }],
-    '05rem': ['0.5rem', { lineHeight: '1' }],
-    '100rem': ['1rem', { lineHeight: '1' }],
-    '110rem': ['1.1rem', { lineHeight: '1' }],
-    '120rem': ['1.2rem', { lineHeight: '1' }],
-    '130rem': ['1.3rem', { lineHeight: '1' }],
-    '140rem': ['1.4rem', { lineHeight: '1' }],
-    '150rem': ['1.5rem', { lineHeight: '1' }],
-    '160rem': ['1.6rem', { lineHeight: '1' }],
-    '170rem': ['1.7rem', { lineHeight: '1' }],
-    '175rem': ['1.75rem', { lineHeight: '1' }],
-    '180rem': ['1.8rem', { lineHeight: '1' }],
-    '190rem': ['1.9rem', { lineHeight: '1' }],
-    '200rem': ['2rem', { lineHeight: '1' }],
-    '210rem': ['2.1rem', { lineHeight: '1' }],
-    '220rem': ['2.2rem', { lineHeight: '1' }],
-    '225rem': ['2.25rem', { lineHeight: '1' }],
-    '230rem': ['2.3rem', { lineHeight: '1' }],
-    '240rem': ['2.4rem', { lineHeight: '1' }],
-    '225rem': ['2.5rem', { lineHeight: '1' }]
+    '16': '1.6rem',
+    '17': '1.7rem',
+    '18': '1.8rem',
+    '19': '1.9rem',
+    '20': '2rem',
+    '21': '2.1rem',
+    '22': '2.2rem',
+    '23': '2.3rem',
```

Comment:
My thinking is that these are for places where you need strict font sizes (usually on the smaller end). For larger fonts, we have the modular typography classes .su-type90 to .su-type-6

Reply:
I also leave line height out of the font sizes since we have font size utility classes.